pkg/raindrop/client.go

```
@@ -581,16 +601,18 @@ func (c *Client) GetAuthorizationCodeHandler(w http.ResponseWriter, r *http.Requ
 
 	code, err := c.GetAuthorizationCode(r)
 	if err != nil {
-		ePrintln(err.Error())
+		c.logger.Error("Authorization error", "error", err)
 		w.WriteHeader(http.StatusUnauthorized)
-		_, err = fmt.Fprintf(w, "<h1>Authorization Error</h1><code>%s</code>", err.Error())
+		if _, writeErr := fmt.Fprintf(w, "<h1>Authorization Error</h1><code>%s</code>", err.Error()); writeErr != nil {
```

Explicitly ignore this error with an `_`.